### PR TITLE
Feature/better animation student reaction

### LINKED ISSUE #356: Spięcie wysyłania poprawnych odpowiedzi z backiem po stronie studenta i wykładowcy
zmień nazwę na "wyślij poprawne odpowiedzi"

### PR DESCRIPTION
**Pull request by ostatni5**

### CODE REVIEW

File: src/lecturer/components/quizStatsView/QuestionBlock.tsx [hunk 1]
```diff
@@ -39,6 +39,7 @@ export function QuestionBlock(props: QuestionBlockProps) {
             },
         },
     })();
+
```
Comment (Petros9) [inline src/lecturer/components/quizStatsView/QuestionBlock.tsx:42]: unnecessary change

**Review by Petros9 — CHANGES_REQUESTED** [review]
the rest seems alright

File: src/lecturer/components/quizStatsView/QuestionBlock.tsx (hunk 1 above)
Comment (ostatni5) [reply src/lecturer/components/quizStatsView/QuestionBlock.tsx:42]: enter is necessary

Comment (rozchlastywacz) [reply src/lecturer/components/quizStatsView/QuestionBlock.tsx:42]: image

**Review by rozchlastywacz — APPROVED** [review]
gg wp

**Review by Petros9 — APPROVED** [review]
glorious

**Pull request merged**